server/subscribe.go

```diff
@@ -714,11 +715,19 @@ func (p *Plugin) httpSubscribeWebhook(w http.ResponseWriter, r *http.Request, in
 		return respondErr(w, http.StatusForbidden,
 			fmt.Errorf("JIRA plugin not configured correctly; must provide Secret"))
 	}
+
 	status, err = verifyHTTPSecret(conf.Secret, r.FormValue("secret"))
 	if err != nil {
 		return respondErr(w, status, err)
 	}
 
+	if conf.EnableWebhookEventLogging {
+		parsedRequest, eventErr := httputil.DumpRequest(r, true)
+		if eventErr != nil {
+			return respondErr(w, status, eventErr)
+		}
+		p.API.LogInfo("Webhook Event Log", "event", string(parsedRequest))
```

Comment:
This needs to change to `LogDebug` too

```suggestion
		p.API.LogDebug("Webhook Event Log", "event", string(parsedRequest))
```